css/DayPickerNavigation.scss

```diff
@@ -0,0 +1,96 @@
+@import 'variables';
+
+.DayPicker__nav--prev,
+.DayPicker__nav--next {
+  cursor: pointer;
+  line-height: 0.78;
+  user-select: none;
+  -webkit-user-select: none; /* Chrome/Safari */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* IE10+ */
+}
+
+.DayPicker__nav-prev-default,
+.DayPicker__nav-next-default {
+  border: 1px solid $react-dates-color-border-light;
+  background-color: $react-dates-color-white;
+  color: $react-dates-color-placeholder-text;
+
+  &:focus,
+  &:hover {
+    border: 1px solid $react-dates-color-border-medium;
+  }
+
+  &:active {
+    background: darken($react-dates-color-white, 5%);
+  }
+}
+
+.DayPicker--horizontal {
+  .DayPicker__nav {
+    position: relative;
+  }
+
+  .DayPicker__nav--prev,
+  .DayPicker__nav--next {
+    border-radius: 3px;
+    padding: 6px 9px;
```

Comment:
I think the `border-radius` should be moved. Not sure about the padding though as I think the position might look weird without this. I'll look into it though.